moped-database/migrations/1668207700121_add_new_phase_cols_to_list_view/up.sql

```diff
@@ -46,11 +46,7 @@ AS WITH project_person_list_lookup AS (
     mp.task_order,
     mp.updated_at,
     current_phase.phase_name as current_phase,
-    -- use of coalesce is a hack to work around hasura's '_neq' filter behavior, which
-    -- would exclude projects with a `null` current phase. We need to patch to the
-    -- GQLAbstract to support a more complex `where` directive. see 'Caveat for null values':
-    -- https://hasura.io/docs/latest/queries/postgres/query-filters/#comparison-operators
-    COALESCE(current_phase.phase_key, '') as current_phase_key,
+    current_phase.phase_key as current_phase_key,
```

Comment:
This change causes canceled projects to be included in the project list. There are currently two of them in production.

This feels right - a canceled project may have relevant project history that is presumably important for users to access for historical purposes.

Reply:
I agree, canceled projects shouldn't be treated the same as deleted projects (hidden from view) 

now im curious to check if canceled projects show up in search results